.github/runs-on.yml

```diff
@@ -1,10 +1,14 @@
 runners:
   self-hosted-ubuntu-22.04-x86-64:
-    cpu: [16, 32, 64]
-    ram: [64, 128, 192]
+    # A range from lowest to highest
+    cpu: [16, 64]
```

Comment:
Not sure if this is a range, from what I read it's an array of values?
https://runs-on.com/configuration/repo-config/#runnerscpu

Reply:
The detailed docs (and the maintainer) say it's a range:
> Number of vCPUs to request (default: 2). If you set multiple values, RunsOn will request any instance matching the lowest up to the highest value.

See the examples here for more context:
https://runs-on.com/configuration/job-labels/#cpu

Reply:
Oh good to know, somehow it worked the other way too